.github/workflows/build_wheels.yml

```diff
@@ -61,20 +65,20 @@ jobs:
     needs: [build_wheels, build_sdist]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
 
       - name: install requirements
         run: pip install twine==3.4.2
```

Comment:
Nitpick: could update twine to `4.0.2`. The main breaking change from v3 to v4 was dropping support for python 3.6.